kernel/devices/Rtc.cpp

```diff
@@ -0,0 +1,62 @@
+#include <devices/Rtc.h>
+#include <Cpu.h>
+#include <Log.h>
+
+namespace Kernel::Devices 
+{
+
+    uint8_t readRtcRegister(uint8_t rtcRegister)
+    {
+        CPU::PortWrite8(CMOS_ADDRESS_REGISTER, rtcRegister);
+        return CPU::PortRead8(CMOS_DATA_REGISTER);
+    }
+    
+    uint64_t readRtcTime() 
```

Comment:
in theory i think we should return a time_t value, do we already have it? or i should define it (apparently is not part of the freestanding)

Reply:
We don't have a time_t or similar, I'm planning to add a DateTime class, similar to how C# does it. I'd like to keep the internal format as unix epoch (a single uint64_t) for now, since it's just a single simple value.